website/server/libs/setupPassport.js

```diff
@@ -1,8 +1,10 @@
 import passport from 'passport';
 import nconf from 'nconf';
 import passportFacebook from 'passport-facebook';
+import passportGoogle from 'passport-google-oauth20';
 
 const FacebookStrategy = passportFacebook.Strategy;
+const GoogleStrategy = passportGoogle.Strategy;
```

Comment:
Why not:

```js
import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
```

Same with the Facebook one, unless `passportFacebook` is used elsewhere.